### PR TITLE
Update v7 CHANGELOG and version

### PR DESCRIPTION
**Pull request by arikfr**
## What type of PR is this? (check all applicable)

- [x] Other

## Description

Added CHANGELOG for v7 changes & updated version string.

This is pending on one more pull request which fixes a bug with dropdown values.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,385 +1,456 @@
 # Change Log
 
+## v7.0.0 - 2019-03-17
+
+We're trying a new format for the CHANGELOG in this release. Focusing on the bigger changes, but for whoever interested, you can see all the changes [here](https://github.com/getredash/redash/compare/v6.0.0...master).
+
+Besides all the features, bug fixes and improvements listed below we managed to convert a large portion of Redash's frontend code from Angular.js to React. You can see status in #3071.
```
Comment (ranbena) [inline CHANGELOG.md]:
```suggestion
Besides all the features, bug fixes and improvements listed below we managed to convert a large portion of Redash's frontend code from Angular.js to React. You can see status in [#3071](https://github.com/getredash/redash/issues/3071).
```

In the preview it doesn't get automatically linked 🤷‍♂️

**Comment by ranbena** [comment]
@arikfr wdyt about images/gifs for FE features?

**Comment by arikfr** [comment]
@ranbena once I create some for the forum post / blog post, I will add them in the CHANGELOG.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,385 +1,456 @@
...
+
+### In-app Help
+
+You can now open the [Knowledge Base](https://redash.io/help) inside the application. We also added a few "help triggers" in the app, that will open the Knowledge Base in context of what you're currently doing.
+
+### Parameters
+
+- **Dashboard Parameters**: We improved the flow of adding queries with parameters to dashboards and now give you full control over how parameters are mapped. You no longer have to make sure all parameters have the same name or use the `Global` checkbox. We also added new options, like keeping the parameter local to the widget or setting a static value. [Read more in our Knowledge Base →](https://redash.io/help/user-guide/querying/query-parameters#Parameter-Mapping-on-Dashboards)
+- We added server side validation of parameter values for all parameter types, except for parameters of `text` type. All validated parameter types are considered safe. When a query is using safe parameters (or no parameters at all), View Only users can refresh it.
+- Refresh of safe queries is done using the new results API endpoint, which takes only a query ID (and optionally parameter values) and does not need the query text.
```
Comment (ranbena) [inline CHANGELOG.md]:
```suggestion
- Refreshing safe queries is done using the new results API endpoint, which takes only a query ID (and optionally parameter values) and does not need the query text.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,385 +1,456 @@
...
+
+### Query Editor Improvements
+
+- Run only the highlighted query text: hit Execute after highlighting a portion of your query and only the selected portion will be sent to the database. This is useful for testing sub-SELECT statements and CTE's.
+- Improved auto complete: add a dot . after a table name in the query editor and auto complete will only suggests columns on that table.
```
Comment (ranbena) [inline CHANGELOG.md]:
```suggestion
- Improved auto complete: add a dot . after a table name in the query editor and auto complete will only suggest columns on that table.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,385 +1,456 @@
...
+- Autosave parameter configuration changes.
+- YAML syntax support (for data sources like Yandex Metrica).
+
+### Improved Query Scheduler
+
+The Query Scheduler got a face lift and some new options: you can pick the day a weekly schedule runs on and also set an end date after which the query will no longer execute on schedule.
```
Comment (ranbena) [inline CHANGELOG.md]:
```suggestion
The Query Scheduler got a face lift and some new options: you can pick a day for a weekly schedule to run on and also set an end date after which the query will no longer execute on schedule.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,385 +1,456 @@
...
+
+### Data Sources
+
+We added Apache Drill, Uptycs and a new JSON data source. Also fixed a few bugs in Athena's query runner and others.
+
+### Users Management
```
Comment (ranbena) [inline CHANGELOG.md]:
```suggestion
### User Management
```

@arikfr I'm never truly certain about this..

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,385 +1,456 @@
...
+
+The users page got revamped with a new look and feel and new features:
```
Comment (ranbena) [inline CHANGELOG.md]:
```suggestion
The users page got revamped with a new look and feel and few new features:
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,385 +1,456 @@
...
+
+- An indication when the user was last active.
```
Comment (ranbena) [inline CHANGELOG.md]:
```suggestion
- An indication when a user was last active.
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,385 +1,456 @@
...
+- Show if an invited user hasn't finished the setup process yet (Pending Invitations section).
+- You can now generate a new API key for users, if there's a concern it was compromised.
+
+### Admin
+
+- New Celery queues status screens to replace the old Queries Status, which should better reflect the status of running queries.
```
Comment (ranbena) [inline CHANGELOG.md]:
```suggestion
- New Celery queues status screens, replacing the old Queries Status and better reflecting the status of running queries.
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,385 +1,456 @@
...
+- Make the queue name for schema refresh job configurable. The default used to be hard coded `schemas`, which is not available on all setups. Now it's `celery`.
+- The `gevent` library is installed by default, and you can setup gunicorn to use `gevent` based workers.
```
Comment (ranbena) [inline CHANGELOG.md]:
```suggestion
- The `gevent` library is installed by default, and you can now setup gunicorn to use `gevent` based workers.
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,385 +1,456 @@
...
+- New Docker entrypoint command to do a health check for a worker process.
+- Flask-Admin is no longer setup or supported.
+
+### Other Changes
+
+- New Alert destination: Google Hangouts Chat.
+- When downloading results from the results API it will set a user friendly filename for the downloaded file.
+- Add Archived Queries section to queries list.
```
Comment (ranbena) [inline CHANGELOG.md]:
```suggestion
- Archived Queries section added to the queries list.
```

**Comment by ranbena** [comment]
@arikfr I'm done proofing

**Comment by arikfr** [comment]
@ranbena thanks!

**Pull request merged**